internal/worker/uniter/runner/context/env_test.go

```diff
@@ -379,13 +255,15 @@ func (s *EnvSuite) TestHostEnv(c *gc.C) {
 	unit := api.NewMockUnit(ctrl)
 	unit.EXPECT().Tag().Return(names.NewUnitTag("this-unit/123")).AnyTimes()
 
-	s.PatchValue(&jujuos.HostOS, func() ostype.OSType { return ostype.GenericLinux })
+	s.PatchValue(&jujuos.HostOS, func() ostype.OSType { return ostype.Ubuntu })
```

Comment:
Why are we doing this? The test is run on Ubuntu. The only reason it was patched was to simulate centos.
Do we even need the ability to patch HostOS any more?

Reply:
If someone is running a test on a mac, HostOS will return MacOSX. So unfortunately we still need this patching